WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1454,6 +1471,10 @@ decisions = {
 			}
 			modifier = {
 				factor = 0
+				trait = tolerant_ruler
```

Comment:
Part of being tolerant means NOT banishing Jews. That's the OPPOSITE of tolerant.